package.json

```diff
@@ -4,8 +4,8 @@
   "description": "game off 2017",
   "main": "src/index.js",
   "scripts": {
-    "build": "webpack",
-    "start": "webpack && webpack-dev-server --port=3030"
+    "build": "nps build",
+    "start": "nps start"
```

Comment:
This change is of little-utility in this PR, but I think `nps` is a great way of encapsulating the build steps cleanly as things become more complex.